api/Screen.json

```diff
@@ -144,6 +177,39 @@
             "standard_track": false,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": false
+            }
+          }
         }
       },
       "availWidth": {
```

Comment:
Should edge be mirror here? (line 224)

Reply:
The trouble here is that if we mirror it, it will show the equivalent to Chrome 1, which is wrong, as it wasn't based on Chromium at that time. I also tried adding in an extra piece of support data to show that later on, Edge 100 (equivalent to Chrome 100) supports this without the note, but then it wants me to call the earlier support a partial implementation. But that's not really right either, as the old Edge implementation was correct that the time.

So I've opted to just leave it alone.